src/filters/normalizeIconClass.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module("openshiftCommonUI")
+  .filter("normalizeIconClass", function() {
+    return function(iconClass) {
+      // if iconClass starts with "icon", append "icon-font "
+      // so the Openshift Logos Icon font is used
+      if(_.startsWith(iconClass, "icon")) {
```

Comment:
Should we be checking for "icon" or "icon-"? I believe all of the OpenShift logos start with "icon-"

Reply:
It might be good to keep the hyphen to avoid false positives